.github/workflows/rust.yml

```
@@ -297,3 +297,29 @@ jobs:
           cargo run -p boa_examples --bin 2>&1 \
             | grep -E '^ ' \
             | xargs -n1 sh -c 'cargo run -p boa_examples --profile ci --bin $0 || exit 255'
+
+  runSemVerCheck:
```

```suggestion
  run-semver-check:
```
To be consistent with the rest of the job names.